### ISSUE TITLE
Restructure the repository branches

### ISSUE THREAD

**Issue by Justus-Ortlepp-frms** [frmscoe/General-Issues#127]
A repository should only contain the following:

1. A `main` branch where the most recent iteration of approved code lives
2. A `ver9.9.x` branch for servicing/patching of a related release "tag"
3. A `developer` branch (explicitly described) that will be created in support of a Pull Request and will be destroyed once the code from that branch is merged into `main`

**Comment by Justus-Ortlepp-frms** [comment]
Automatically delete DependaBot branches when the PR is accepted. Robotic branches may need to be cleaned up periodically.

**Comment by Sandy-at-Tazama** [comment]
Marked as done, can be closed
